test/data/manifests/rhel_90-x86_64-ec2-boot.json

```diff
@@ -2360,7 +2140,7 @@
                 "origin": "org.osbuild.pipeline",
                 "references": {
                   "name:image": {
-                    "file": "image.raw"
```

Comment:
Yes, seeing how everything works now, it might be worth having a way to specify the base filename separately from the final artifact filename in cases where an image is compressed or tarred.